pkgs/development/python-modules/pyjson/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyjson";
+  version = "1.4.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "leeyoshinari";
+    repo = "Small_Tool";
+    rev = "refs/tags/${version}";
+    hash = "sha256-tsdRUp7LoyJINrdqiJ7K4Oi//LNUBK2Og9OuDFZxl0k=";
+  };
+
+  sourceRoot = "${src.name}/pyjson";
+
+  build-system = [ setuptools ];
+
+  doCheck = false; # no tests
+
+  pythonImportsCheck = [ "pyjson" ];
+
+  meta = {
+    changelog = "https://github.com/leeyoshinari/Small_Tool/releases/tag/${version}";
+    description = "Compare the similarities between two JSONs";
+    homepage = "https://github.com/leeyoshinari/Small_Tool";
```

could be
```suggestion
    homepage = "https://github.com/leeyoshinari/Small_Tool/tree/master/pyjson";
```
but that's optional.